application/templates/alertmanagerconfig.yaml

```diff
@@ -1,13 +1,11 @@
-{{- if .Values.alertmanagerConfig -}}
-{{- if .Values.alertmanagerConfig.enabled -}}
+{{- if and (.Values.alertmanagerConfig).enabled (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1alpha1") -}}
 apiVersion: monitoring.coreos.com/v1alpha1
 kind: AlertmanagerConfig
 metadata:
   name: {{ template "application.name" . }}
-  namespace: {{ template "application.namespace" . }}
+  namespace: {{ include "application.namespace" . }}
   labels:
-{{ include "application.labels.stakater" . | indent 4 }}
-{{ include "application.labels.chart" . | indent 4 }}
+{{ include "application.labels" . | indent 4 }}
```

Comment:
we can do `{{- include "application.labels" . | nindent 4 }}`  and indent this line

Reply:
we should indent in all files; @mustafaStakater can you do that?

Reply:
fixed